domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraUseCase.kt

```diff
@@ -73,6 +73,8 @@ interface CameraUseCase {
 
     fun getZoomScale(): StateFlow<Float>
 
+    fun getLowLightBoostActiveStatus(): StateFlow<Boolean>
```

Comment:
Have you considered defining an enum for this? Maybe `UNSUPPORTED, ACTIVE, INACTIVE` ?
That way the client only needs to observe this and not `isLowLightBoostSupport` and the default can be `UNSUPPORTED` rather than `true`